### DISCUSSION TITLE
Jump button in view mode?

### DISCUSSION THREAD

**Discussion by pudah** [gsantner/markor#1891] [Q&A] [answered]
Hello everyone. Quick question.

In the recent versions of Markor, there is a tool bar at the bottom of the screen when in view mode.  Am I correct in my assumption that the left-most button (line with down / up arrows above and below) is to enable the user to quickly jump to the top or bottom of the file? 

I ask because it does not do that when I tap it. So it makes me doubt whether or not I understand the purpose of the button.

**Comment by gsantner** [comment, marked as answer]
Yes this is alright, correct. That button is to jump to top/bottom.

And it works perfectly fine here at everything I tried so far, like you can see below.

[screenrecord-2022-10-12_00.29.37.webm](https://user-images.githubusercontent.com/6735650/195210195-6d901ef1-8b87-4833-9467-ecedea28f464.webm)

But there might be a case that I don't know where it not works. If you have issues, feel free to fix it in a merge request.  It's Hacktoberfest and the best time of the year to start contributing to Open Source projects - https://github.com/gsantner/markor/issues/1449 .

**Reply by pudah** [reply]
Thanks for the quick reply.

I think it would be great to contribute to an open source project. 
I was a mainframe assembler and COBOL programmer for 30+ years. But sadly I don't know anything about JAVA or Android development. I don't know that you'd want someone with no knowledge or experience mucking about in your code 😀

**Reply by gsantner** [reply]
> ut sadly I don't know anything about JAVA or Android

Never too late to learn something new.

> I don't know that you'd want someone with no knowledge or experience mucking about in your code

I do,  any helping hand is appreciated - there are not much of them.

**Reply by harshad1** [reply]
@pudah FWIW, This is the _only_ Android / Java codebase I have ever contributed to.

**Reply by pudah** [reply]
> @pudah FWIW, This is the _only_ Android / Java codebase I have ever contributed to.

@harshad1 did you already know java or did you learn so that you could contribute to markor?

If the latter, can you share how you got started?

**Reply by harshad1** [reply]
I didn't really know java. I am a professional software engineer with lots of experience with c++ etc though, so I just dove in and used stackoverflow + google as one does. 

IMO you should pick a task and get started. If you run into a problem, google + stack overflow will tell you how to solve the problem idiomatically. It will be slow at first, but eventually you will build up a feel for Markor's codebase.
